packages/pigeon/platform_tests/alternate_language_test_plugin/android/src/main/java/com/example/alternate_language_test_plugin/CoreTests.java

```diff
@@ -65,6 +70,10 @@ protected static FlutterError createConnectionError(@NonNull String channelName)
         "channel-error", "Unable to establish connection on channel: " + channelName + ".", "");
   }
 
+  @Target(METHOD)
+  @Retention(CLASS)
+  @interface CanIgnoreReturnValue {}
```

Comment:
Thoughts about also emitting a `CheckReturnValue` annotation on the enclosing type? (That also means generating a CRV annotation as well)

Reply:
It won't help internal Google users (we have CRV by default), but any external users who are using static analysis would definitely benefit the addition of `@CheckReturnValue` on the enclosing type. (That being said, it can be done later, so a TODO is also fine.)

Reply:
I think we'll wait for now. If there are users who are interested in this being added, I encourage you to file an issue (or throw up a quick pr!).